src/apps/voting/hooks/useVotes.js

```diff
@@ -1,20 +1,22 @@
 import { useMemo } from 'react'
-import { useCurrentApp, useInstalledApps } from './shared'
+import { useOrganizationState } from '@/providers/OrganizationProvider'
+import { useApp } from '@/hooks/shared'
+import { EMPTY_ADDRESS, shortenAddress } from '@/utils/web3-utils'
 import { useAppState } from '../providers/VotingProvider'
 import { isVoteOpen } from '../vote-utils'
 import { VOTE_ABSENT } from '../vote-types'
-import { EMPTY_ADDRESS, shortenAddress } from '../web3-utils'
-import useNow from './useNow'
+import useNow from '@/hooks/shared/useNow'
 
 // Decorate the votes array with more information relevant to the frontend
 function useDecoratedVotes() {
   const { votes, connectedAccountVotes } = useAppState()
-  const currentApp = useCurrentApp()
-  const installedApps = useInstalledApps()
+  // TODO: get app by address instead of name
+  const { apps } = useOrganizationState()
```

Comment:
ahhh nice catch with this!! totally forgot to remove this dummy function that sem created while ago when making the firtst version